public/javascripts/itemDetailView.js

```diff
@@ -24,11 +31,9 @@ var itemDetailView = {
             } else {
                 self.renderError("No inventory found");
             }
-        })/*.fail(function (response) {
-            self.renderError("Failed to load inventory: " + response);
-        })*/;
+        });
 
-        navigation.hit("/GetRunsByProduct/" + this.productID, function (response) {
+        navigation.get("/GetRunsByProduct/" + this.productID, function (err, response) {
```

Comment:
No, the navigation functions I added are robust enough to add it if it is needed:
https://github.com/johnschroeder/team-c-front-end/blob/IM-199-Navigation-Hit-Feature/public/javascripts/navigation.js#L64